### PR TITLE
Features/three score

### PR DESCRIPTION
**Pull request by zbrotherton**
Two score needs to be merged before this PR is reviewed

### CODE REVIEW

File: src/main/java/frc/robot/classes/Auton.java [hunk 1]
```diff
@@ -500,6 +515,7 @@ private Command getAutonEnd(){
     }
 
     public Command getAutonCommand(){
-        return getAutonSequence().beforeStarting(delay(delayEntry.getDouble(0.0))).andThen(getAutonEnd());
+        SmartDashboard.putString("auton", autonMode.toString());
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java]: Is this for debug? If not, we should put it on a tab

File: src/main/java/frc/robot/classes/Auton.java [hunk 2]
```diff
@@ -99,8 +103,13 @@ public Auton(SwerveSubsystem swerveSubsystem, ArmSubsystem armSubsystem, IntakeS
         actions.put("armConeIntake", armSubsystem.transitionToState(ArmState.CONEINTAKE));
         actions.put("armCubeIntake", armSubsystem.transitionToState(ArmState.CUBEINTAKE));
         actions.put("cubePullTransition", intakeSubsystem.pullTimed(1.5, false).andThen(highTransitionSequenceCube()));
+        actions.put("cubePullIntake", intakeSubsystem.pullTimed(1.5, false).andThen(armSubsystem.transitionToState(ArmState.STOWED)));
+        actions.put("cubeScore", scoreSequenceCube());
         actions.put("conePull", intakeSubsystem.pullTimed(1.5, true).andThen(armSubsystem.goToState(ArmState.STOWED)));
         actions.put("conePullTransition", intakeSubsystem.pullTimed(1.5, true).andThen(highTransitionSequenceCone()));
+
+        autonCommand = getAutonSequence();
+        configTab.add(new InstantCommand(() -> {autonCommand = getAutonSequence();}).beforeStarting(new WaitCommand(1)).withName("Load Auton").ignoringDisable(true));
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java]: Add comments about why the WaitCommand is included

File: src/main/java/frc/robot/classes/Auton.java [hunk 3]
```diff
@@ -28,6 +28,7 @@
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.StartEndCommand;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java]: Delete, no longer used

File: src/main/java/frc/robot/classes/Auton.java [hunk 4]
```diff
@@ -78,6 +79,9 @@
     private PathConstraints pathConstraints;
 
     private AutonModes autonMode;
+    private Command autonCommand;
+    private Trigger modeTrigger;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java]: Delete, no longer used

**Review by azvanderpas — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 5]
```diff
@@ -28,7 +30,7 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-  public static final boolean isCompetition = true;
+  public static final boolean isCompetition = false;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Robot.java:31]: Revert

File: src/main/java/frc/robot/Robot.java [hunk 6]
```diff
@@ -18,8 +18,10 @@
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.PowerDistribution;
 import edu.wpi.first.wpilibj.PowerDistribution.ModuleType;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Robot.java]: Are these additions used?

File: src/main/java/frc/robot/classes/Auton.java (hunk 1 above)
Comment (azvanderpas) [reply src/main/java/frc/robot/classes/Auton.java]: And if it is for debugging, we should removed it...

File: src/main/deploy/pathplanner/generatedCSV/1Score.csv
Comment (azvanderpas) [inline src/main/deploy/pathplanner/generatedCSV/1Score.csv:1]: Remove the generatedCSV folder and add it to the .gitignore

**Review by azvanderpas — CHANGES_REQUESTED** [review]

File: src/main/deploy/pathplanner/generatedCSV/1Score.csv
Comment (Shazamboo) [reply src/main/deploy/pathplanner/generatedCSV/1Score.csv:1]: done

**Review by azvanderpas — APPROVED** [review]

**Pull request merged**